### DISCUSSION TITLE
Tidak bisa run app

### DISCUSSION THREAD

**Discussion by 19807311** [teamalgoritma/community#414] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/114885734/198188697-28893aef-46ea-48cc-ab2e-9690b022f157.png)

Halo mau tanya kalau ada seperti ini di console (tidak bisa run app) artinya kenapa ya?

**Comment by onlyphantom** [comment, marked as answer]
App nya memanggil API dari github dan kuota API nya udah habis. 

> Only -1 Github queries remaining until 2022-10-27

Jadi sampai tanggal 27th pukul 11:35:09 nanti baru kuota nya di-refresh. Sampai 11:35 API nya sudah tidak bisa dipanggil. Suggestion nya di sign up buat akun untuk mendapatkan API rate yang lebih tinggi.

**Reply by 19807311** [reply]
thank you
